website/src/Components/AccountComponents/StateManager.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import TournamentViewPage from '../PageComponents/TournamentViewPage';
+import TournamentCreatePage from '../PageComponents/TournamentCreatePage';
+import TournamentRulesPage from '../PageComponents/TournamentRulesPage';
+import TeamsViewPage from '../PageComponents/TeamsViewPage';
+import TeamsCreatePage from '../PageComponents/TeamsCreatePage';
+import TeamsJoinPage from '../PageComponents/TeamsJoinPage';
+import TeamsAboutPage from '../PageComponents/TeamsAboutPage';
+import HomePage from '../PageComponents/HomePage';
+import LoginPage from '../PageComponents/LoginPage';
+import AccountCreationPage from '../PageComponents/AccountCreationPage';
+
+
+class StateManager extends Component{
+	constructor(){
+		super();
+		this.state = {
+			loggedIn: false,
```

Comment:
I think this whole class is the source of your login persistence issue. You should not store being logged in on the front end. You should just attempt to retrieve data from the server and if you're not logged in the backend will tell you in an error response. When you do it this way, whenever the user refreshes, the loggedIn value will be set back to false.